### ISSUE TITLE
Would you have any plans to release the pretrained model?

### ISSUE THREAD

**Issue by JiahaoXia** [UniBwTAS/YOLOPoint#1]
Hi, Thanks for your great work.
I am curious if you have any plans to release the pre-trained model's weight.
It would be great to test your model in other scenarios.

**Comment by antopost** [comment]
Yes! And sorry for the late reply. 
I have uploaded some weights for COCO and KITTI to huggingface. 
You can now find the download links in the README.md.
Additionally, I have been experimenting with new architectures, loss functions and augmentation strategies but haven't thoroughly evaluated the resulting models.
If you'd like to check them out I have also provided links to the weights of the experimental models.
